cmd/cluster/context.go

```diff
@@ -393,6 +400,25 @@ func (o *contextOptions) generateContextData() (*contextData, []error) {
 		}
 	}
 
+	GetDynatraceURL := func() {
+		defer wg.Done()
+		if o.verbose {
+			fmt.Fprintln(os.Stderr, "Getting Dynatrace URL...")
+		}
+		if !strings.HasPrefix(cluster.Name(), "hs-mc-") {
```

Comment:
We can call the OCM cluster api and check the labels on the cluster to see whether it's MC or not.
/api/clusters_mgmt/v1/clusters/<cluster_id>/external_configuration/labels

Reply:
ack.
I will try to improve the check in that way

Reply:
Updated with same